src/smart-components/user/user-table-helpers.tsx

```diff
@@ -29,47 +31,60 @@ export interface RowProps {
     CellObject // status
   ];
   selected: boolean;
+  advancedPermissions?: boolean;
 }
 
-export const createRows = (userLinks: boolean, data: UserProps[] = [], intl: IntlShape, checkedRows = [], isSelectable = false): RowProps[] =>
-  data?.reduce<RowProps[]>((acc, { username, is_active: isActive, email, first_name: firstName, last_name: lastName, is_org_admin: isOrgAdmin }) => {
-    const newEntry: RowProps = {
-      uuid: username,
-      cells: [
-        isOrgAdmin ? (
-          <Fragment>
-            <CheckIcon key="yes-icon" className="pf-v5-u-mr-sm" />
-            <span key="yes">{intl.formatMessage(messages.yes)}</span>
-          </Fragment>
-        ) : (
-          <Fragment>
-            <CloseIcon key="no-icon" className="pf-v5-u-mr-sm" />
-            <span key="no">{intl.formatMessage(messages.no)}</span>
-          </Fragment>
-        ),
-        {
-          title: userLinks ? (
-            <AppLink to={pathnames['user-detail'].link.replace(':username', username)}>{username.toString()}</AppLink>
+export const createRows = (
+  userLinks: boolean,
+  data: UserProps[] = [],
+  intl: IntlShape,
+  checkedRows = [],
+  isSelectable = false,
+  advancedPermissions?: boolean
```

Comment:
Let's rename it to `authModel`.
```suggestion
  authModel?: boolean
```

Reply:
Fixed